desktop/core/src/desktop/js/reactComponents/FileChooser/FileChooserModal/FileChooserModal.tsx

```diff
@@ -127,8 +127,10 @@ const FileChooserModal: React.FC<FileProps> = ({ show, onCancel, title, okText }
               <Row className="hue-path-browser-panel" onClick={e => e.stopPropagation()}>
                 <Col span={18}>
                   <PathBrowser
-                    handleFilePathChange={setFilePath}
                     breadcrumbs={filesData?.breadcrumbs}
+                    handleFilePathChange={setFilePath}
```

Comment:
I would name the prop **on**FilepathChange because it is used to let another component react to an event. The function actually handling the event could be called handleFilePathChange (or setFilePath like you used here)